src/runtime_src/core/pcie/driver/linux/xocl/subdev/xgq.c

```diff
@@ -2614,6 +2716,14 @@ static int xgq_vmr_probe(struct platform_device *pdev)
 		return ret;
 	}
 
+	//VMR Identify Command
+        ret = xgq_vmr_identify_cmd(pdev);
+        if(ret)
+	{
```

Comment:
great, we might just fail here, we need more time to think though if marjor.minor is not satisficated what we should do here. I will bring this topic to the XGQ related team for discussion.